pycobertura/cobertura.py

```diff
@@ -58,6 +67,17 @@ def __init__(self, report, filesystem=None):
     def __eq__(self, other):
         return self.report and other.report and self.report == other.report
 
+    def _load_from_path(self, report_path):
+        # it's not a xml-string, so it should be a path.
+        # is it existed?
```

Comment:
Is this comment relevant here? Maybe we can remove it, the code is pretty self-explanatory to me.

```suggestion
```